### ISSUE TITLE
Support for casbin 2

### ISSUE THREAD

**Issue by kuchaguangjie** [maxwellhertz/gin-casbin#1]
When use with `casbin 2`, get error:

> interface conversion: *gormadapter.Adapter is not persist.Adapter: missing method LoadPolicy

Seems, currently this project only support casbin **1**, not **2**.

**Comment by hsluoyz** [comment]
@closetool

**Comment by kuchaguangjie** [comment]
@maxwellhertz Could you make a release, e.g `v2.0.0`, so  that I could test it.

**Comment by kuchaguangjie** [comment]
I have finished the testing, by forking / release / replace in go.mod.
The new code works well, with casbin 2.x.

Tag of fork: https://github.com/kuchaguangjie/gin-casbin/releases/tag/v1.1.0
require statement in go.mod:
> replace github.com/maxwellhertz/gin-casbin => github.com/kuchaguangjie/gin-casbin v1.1.0

Tips:
 - I avoided v2 release, cause that seems need to copy go.mod into `v2/` sub dir, which is the correct way if the new api is not backward compatible. But, that's not necessary for my testing.
